site/book/03-packages/06-composing-a-package.md

```diff
@@ -0,0 +1,59 @@
+_Package composition_ is when you change the package hierarchy by adding or removing subpackages.
+There are two different ways to add a subpackage to a package on the local filesystem:
+
+1. [Create a new package] in a subdirectory
+2. [Get an existing package] in a subdirectory
+
+Let's revisit the `wordpress` package and see how it was composed in the first place.
+Currently, it has the following package hierarchy:
+
+```yaml
+$ kpt pkg tree wordpress/
+PKG: wordpress
+├── [Kptfile]  Kptfile wordpress
+├── [service.yaml]  Service wordpress
+├── deployment
+│   ├── [deployment.yaml]  Deployment wordpress
+│   └── [volume.yaml]  PersistentVolumeClaim wp-pv-claim
+└── PKG: mysql
+    ├── [Kptfile]  Kptfile mysql
+    ├── [deployment.yaml]  PersistentVolumeClaim mysql-pv-claim
+    ├── [deployment.yaml]  Deployment wordpress-mysql
+    └── [deployment.yaml]  Service wordpress-mysql
+```
+
+First, let's delete the `mysql` subpackage:
+
+```shell
+rm -r wordpress/mysql
+```
+
+We're going to add back the `mysql` subpackage using the two different ways:
+
+### Create a new package
+
+```shell
+$ cd wordpress
+$ mkdir mysql
+$ kpt pkg init mysql
+# author resources in mysql
+```
+
+This creates a [dependent package].
+
+### Get an existing package
+
+```shell
+$ cd wordpress
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/wordpress/mysql@package-examples/wordpress/v0.1
```

Comment:
I'm not sure if this is the best example here. Would it be better if we pulled in a subpackage from a completely separate repo?